src/docs/cmdstan-guide/running.tex

```diff
@@ -1014,14 +1014,19 @@ \subsection{Variational Inference-Specific Commands}
 %
   \hierlongcmd{\indentarrow}{\farg{\bfseries variational}}
     {Variational inference}
-    {Valid subarguments: \code{algorithm, grad\_samples, elbo\_samples,
-    eta, iter, tol\_rel\_obj, eval\_elbo, output\_samples}}
+    {Valid subarguments: \code{algorithm, iter, grad\_samples, elbo\_samples,
+    eta, adapt, tol\_rel\_obj, eval\_elbo, output\_samples}}
 %
     \hiercmdarg{\indentarrow\indentarrow}{algorithm}{$$list element$$}
       {Variational inference algorithm}
       {Valid values: \ \code{meanfield, fullrank}}
       {Defaults to \code{meanfield}}
 %
+    \hiercmdarg{\indentarrow\indentarrow}{iter}{$$int$$}
+      {Maximum number of iterations}
+      {Valid values: \ $0 < \mbox{\code{iter}}$}
+      {Defaults to \code{10000}}
```

Comment:
That seems like a lot of default iterations for an optimization procedure.  But anyway, we can change these in the future if it turns out to look like it hangs to users.